example/panel/panel_app.dart

```diff
@@ -31,9 +31,9 @@ Future<Null> main() async {
   Element container = querySelector('#panel-container');
   react_client.setClientConfiguration();
 
-  final tracer = new SampleTracer();
-  initGlobalTracer(tracer);
-  assert(globalTracer() == tracer);
+//  final tracer = new SampleTracer();
+//  initGlobalTracer(tracer);
+//  assert(globalTracer() == tracer);
```

Comment:
???

Reply:
The only way to get the example to run properly is to comment those out. I assume someone more knowledgable than me will come in and actually fix it, but the `SampleTracer` is the culprit. See the original screenshot I posted.